src/Sparrow/Memory.cs

```diff
@@ -261,5 +263,22 @@ public static T Read<T>(byte* ptr)
         {
             return Unsafe.Read<T>(ptr);
         }
+
+        [MethodImpl(MethodImplOptions.AggressiveInlining)]
+        public static void Discard(void* baseAddress, long size)
```

Comment:
We should ignore all exceptions here (log this and "disable" the discard for all future calls)

Reply:
We should figure out which OSs do not support it correctly instead. Logging and disabling is not a good idea, because it hides from us important platform assumptions that are broken.